package.json

```diff
@@ -40,6 +42,9 @@
     "@angular/platform-browser": "2.0.0",
     "@angular/platform-browser-dynamic": "2.0.0",
     "@angular/router": "3.0.0",
+    "@ngrx/core": "1.2.0",
+    "@ngrx/effects": "2.0.0-beta.3",
+    "@ngrx/store": "2.2.0",
```

Comment:
Are you kidding me? They are playing with me :smile: I updated this 3 or 4 times now. It's done.